src/WebJobs.Extensions.DurableTask/ContextImplementations/DurableClient.cs

```diff
@@ -563,6 +571,20 @@ async Task<PurgeHistoryResult> IDurableOrchestrationClient.PurgeInstanceHistoryA
             return await this.DurabilityProvider.PurgeInstanceHistoryByInstanceId(instanceId);
         }
 
+        async Task<PurgeHistoryResult> IDurableOrchestrationClient.PurgeInstanceHistoryAsync(IEnumerable<string> instanceIds)
+        {
+            var tasks = instanceIds.Select(instanceId => this.DurabilityProvider.PurgeInstanceHistoryByInstanceId(instanceId));
+            var results = await Task.WhenAll(tasks);
+            var instancesDeleted = 0;
+            foreach (var purgeHistoryResult in results)
+            {
+                    instancesDeleted += purgeHistoryResult.InstancesDeleted;
+            }
+
```

Comment:
nitpick: seems this line was overly-indented.

```suggestion
            var instancesDeleted = 0;
            foreach (var purgeHistoryResult in results)
            {
                instancesDeleted += purgeHistoryResult.InstancesDeleted;
            }

```

Reply:
Thanks for reminding, I didint notice that. I will update it!